index.html

```diff
@@ -105,7 +128,7 @@ <h3 class="text_titleH3">Project 3</h3>
             <div class="container-work" id="project4">
                 <div class="img-work img-work-source4"></div>
                 <div class="div-title flex-work-title">
-                    <h3 class="text_titleH3">Project 4</h3>
+                    <h3 class="text-titleH3">Project 4</h3>
                 </div>
                 <div class="flex-ul-work">
                     <div class="ul-work-item">
```

Comment:
This project width is perfect! 😃

But let's try to make it start at the middle of the page 😁

Reply:
Hello @Kakalanp,

I don't understand how I can make it start in the middle, please could you help me?